CONTRIBUTING.md

```diff
@@ -18,20 +18,24 @@ If you're new to open-source development, we recommend going through the GitHub
 We follow something called a "fork and pull request" workflow for collaborating on our project. See [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) for a great overview on what some of these mysterious terms mean! 
 
 Once you've 'forked' and 'cloned' the code to your local machine, please follow these steps:
+
+```shell
+# change location to the root directory of the superduperdb repository
+cd /path/to/superduperdb/root
+
+# create your Python virtual environment
+python3 -m venv .venv
+
+# activate the Python virtual environment
+. .venv/bin/activate  
 ```
-cd /path/to/superduperdb/root  # change location to the root directory of the superduperdb repository
-```
-```
-python3 -m venv .venv  # create your Python virtual environment
-```
-```
-. .venv/bin/activate  # activate the Python virtual environment
-```
-```
-python3 -m pip install --upgrade pip-tools  # install pip-tools and latest version of pip
-```
-```
-python3 -m pip install -e .[dev]  # Install the superduperdb project in editable mode along with the developer tools
+
+```shell
+# install pip-tools and latest version of pip
+python3 -m pip install --upgrade pip-tools
+
+# Install the superduperdb project in editable mode along with the developer tools
+python3 -m pip install -e '.[dev]'
```

Comment:
use `'.[dev]'` instead of `.[dev]` which can be run in most of operation system

Reply:
interesting